torch/fx/experimental/sym_node.py

```diff
@@ -1109,6 +1133,8 @@ def rbinary_magic_impl(self, other):
 
     if method in unary_magic_methods:
         setattr(user_type, f"__{method}__", unary_magic_impl)
+    elif method in unary_nonmagic_methods:
+        setattr(user_type, method, unary_magic_impl)
```

Comment:
These methods need docs, you could do this by writing a stub in `SymFloat` and use `functolls.wraps` here to take the docs from the stub.